chapters/synchronous.tex

```diff
@@ -127,30 +122,32 @@ \subsection{Rationale for Clocked Semantics}\label{rationale-for-clocked-semanti
     controller part, and different integer multiples are given, then the
     translator has to accept this (no error is detected).
   \end{enumerate}
+  Note: Clocked systems can mix different sampling rates
+  in well-defined ways when needed.
 \item
-  Due to the automatic sample and hold semantics, all variables
-  assigned in a when-clause of the above kind must have an initial value
+  Fewer initial conditions are needed, as only a subset of clocked
+  variables need initial conditions - the clocked state variables \cref{clocked-state-variables}.
+  For a standard when-clause all variables
+  assigned in a when-clause must have an initial value
```

Comment:
The phrase _must have an initial value_ makes me nervous — I mean, don't all variables have initial values?

How about something like this?
> … need to be considered for the initialization problem, as they may be used before they are assigned…